.travis.yml

```diff
@@ -41,6 +41,7 @@ jobs:
         directories:
           - tmp/rubocop_cache
       before_install:
+        # These are needed for the linters to be able to run everywhere
```

Comment:
I would be more concrete with what _everywhere_ means. Something like:

These lines are needed to run rubocop in the root directoy, as the gems are installed in src/api